src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { linkPeople } from '../utils/helpers';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(peopleFromServer => {
+        setPeople(linkPeople(peopleFromServer));
+      })
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+          {isError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+          {(!isLoading && !isError && people && people.length === 0) && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
+          {people && (
+            <PeopleTable people={people} />
+          )}
```

Comment:
```suggestion
          {!!people?.length && (
            <PeopleTable people={people} />
          )}
```
in rendering with &&  need to convert the value to boolean.